cmd/vic-init/ops.go

```diff
@@ -83,6 +86,20 @@ func (t *operations) SetHostname(name string, aliases ...string) error {
 	return t.BaseOperations.SetHostname(aliases[0])
 }
 
+func (t *operations) SetupFirewall() error {
+	// get the public interface name
+	l, err := netlink.LinkByName(publicIfaceName)
+	if l == nil {
+		l, err = netlink.LinkByAlias(publicIfaceName)
+		if l == nil {
+			return fmt.Errorf("could not find public interface")
+		}
+	}
+
+	_, err = iptables.Raw(string(iptables.Append), "FORWARD", "-i", "bridge", "-o", l.Attrs().Name, "-j", "ACCEPT")
```

Comment:
At the end I would prefer to see "dedicated" methods which are adding specific IP tables rules in a little more human way.

iptables.Append(chain, inInterface, outInterface, action string)